apps/challenges/utils.py

```diff
@@ -32,6 +32,13 @@
 get_challenge_phase_split_model = get_model_object(ChallengePhaseSplit)
 
 
+def key_exists_in_json(json, key):
+    try:
+        name = json[key]
```

Comment:
this can be done in better way by getting `json.keys()` and using `key in keys`

Reply:
That way...I guess I'll send all the missing keys at once in a message. The method could return a list.